src/cis/fam_cis_direct.cpp

```diff
@@ -290,29 +290,44 @@ Fam_CIS_Direct::create_region(string name, size_t nbytes, mode_t permission,
     }
 
     std::list<std::shared_future<void>> resultList;
-    try {
     for (auto memsrv : memoryServiceList) {
         Fam_Memory_Service *memoryService = memsrv;
+	size_t size =  nbytes/used_memsrv_cnt;
+	size_t aligned_size = (size + 64 - 1) & -64;
```

Comment:
Use macros and not actual values. Not sure if i completely understood the logic here. We will discuss it tomorrow.